### PR TITLE
ball 1 (new cask)

### PR DESCRIPTION
**Pull request by jrmann100**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
```diff
- GitHub repository too new (<30 days old)
```
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/b/ball.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    url :url
+    regex(/^v(\d+(?:\.\d+)*)$/i)
+    strategy :github_latest do |json, regex|
+      match = json["tag_name"]&.match(regex)
+      next if match.blank?
+
+      match[1]
+    end
+  end
+
```
Comment (bevanjkay) [inline Casks/b/ball.rb:20]:
```suggestion
```

You should be able to omit the `livecheck` stanza, as the default `git` strategy should suffice here.

Comment (jrmann100) [reply Casks/b/ball.rb:20]: Thanks!

**Comment by bevanjkay** [comment]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by jrmann100** [comment]
@bevanjkay, as far as I can tell there isn't anything to zap.

**Review by p-linnane — APPROVED** [review]

**Pull request merged**